src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -234,7 +235,7 @@ def _get_variance(self, t, predicted_variance=None, variance_type=None):
             return predicted_variance
         elif variance_type == "learned_range":
             min_log = torch.log(variance)
-            max_log = torch.log(self.betas[t])
+            max_log = torch.log(current_beta_t)
```

Comment:
when skipping timesteps, self.betas[t] is the wrong beta. see line 211 where we re-calculate the beta for the current timestep and previous timestep

Reply:
Brutal.

Reply:
@sayakpaul two days of debugging 🙃